aws-lc-rs/src/aead/chacha20_poly1305_openssh.rs

```diff
@@ -221,10 +222,20 @@ mod tests {
         let chacha_key = chacha_key.as_slice();
         let chacha_key_bytes: [u8; 32] = <[u8; 32]>::try_from(chacha_key).unwrap();
         let chacha_key = ChaCha20Key::from(chacha_key_bytes);
-        let iv = Nonce::from(&[45u32, 897, 4567]);
-        let poly1305_key = derive_poly1305_key(&chacha_key, iv);
+        {
+            let iv = Nonce::from(&[45u32, 897, 4567]);
+            let poly1305_key = derive_poly1305_key(&chacha_key, iv);
+            assert_eq!(&expected_poly1305_key, &poly1305_key.key_and_nonce);
+        }
 
-        assert_eq!(&expected_poly1305_key, &poly1305_key.key_and_nonce);
+        {
+            let x = LittleEndian::from(45u32);
+            let y = LittleEndian::from(897);
+            let z = LittleEndian::from(4567);
+            let iv = Nonce::from(&[x, y, z]);
+            let poly1305_key = derive_poly1305_key(&chacha_key, iv);
+            assert_eq!(&expected_poly1305_key, &poly1305_key.key_and_nonce);
```

Comment:
Do we need a `BigEndian` version of this test?  Or does that get covered above when we run against the new power pc targets